tools/analyzers/uintcast/analyzer_test.go

```diff
@@ -0,0 +1,12 @@
+package uintcast_test
+
+import (
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/tools/analyzers/uintcast"
+	"golang.org/x/tools/go/analysis/analysistest"
+)
+
+func TestAnalyzer(t *testing.T) {
```

Comment:
Why is this test excluded from Bazel?

Reply:
It doesn't work. It depends on $GOROOT. I left a TODO, ill try to fix it

Reply:
OK. I fixed this and then fixed all of the analyzer tests.